### PR TITLE
SDK CLI: add relevant logs

### PR DESCRIPTION
**Pull request by bzz**
While developing/debugging a new analyzer I found these logs were super helpful in SDK binary.

### CODE REVIEW

File: cmd/lookout/event.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ func (c *EventCommand) openRepository() error {
 }
 
 func (c *EventCommand) resolveRefs() (*lookout.ReferencePointer, *lookout.ReferencePointer, error) {
+	log.Infof("Resolving to/from references")
```
Comment (carlosms) [inline cmd/lookout/event.go]: should this be not capitalized?

File: cmd/lookout/event.go [hunk 2]
```diff
@@ -118,6 +119,7 @@ func (c *EventCommand) makeDataServerHandler() (*lookout.DataServerHandler, erro
 }
 
 func (c *EventCommand) bindDataServer(srv *lookout.DataServerHandler, serveResult chan error) (*grpc.Server, error) {
+	log.Infof("starging a DataServer at %s", c.DataServer)
```
Comment (carlosms) [inline cmd/lookout/event.go]: starting*

File: cmd/lookout/event.go [hunk 3]
```diff
@@ -133,6 +135,7 @@ func (c *EventCommand) bindDataServer(srv *lookout.DataServerHandler, serveResul
 
 func (c *EventCommand) analyzerClient() (lookout.AnalyzerClient, error) {
 	var err error
+	log.Infof("start looking for Analyzer at %s", c.Args.Analyzer)
```
Comment (carlosms) [inline cmd/lookout/event.go]: should this be **starting**, to be consistent with the other logs?

File: cmd/lookout/event.go (hunk 1 above)
Comment (bzz) [reply cmd/lookout/event.go]: seems to be fine, as this is a line-oriented logging, no?

**Comment by bzz** [comment]
Feedback addressed in 9245fc4, ready for another round.

File: cmd/lookout/event.go (hunk 1 above)
Comment (carlosms) [reply cmd/lookout/event.go]: both are fine to me, I was proposing lowercase for consistency with the other ones

**Pull request merged**